Assets/MRTK.ThirdParty/MRTK-Quest/Editor/MRTKQuestConfigurationChecker.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace prvncher.MixedReality.Toolkit.OculusQuestInput
```

Comment:
@keveleigh @davidkline-ms 

As part of bringing this into MRTK mainline, I was wondering if we could just rename this to Microsoft.MixedReality.Toolkit, because, from a consumer perspective, it would be awesome if they didn't have to think about the original origin (or just origin) of where a particular bit of code came from as they were writing code (writing code be like oh yeah I have to user prvncher in this case, hololabs or something else in another case, etc).

Especially if they're not explicitly doing the work here to grab another extension, but are just getting things built into mainline.

Reply:
Also question for you too @RogPodge

Reply:
Though I think it makes sense consumer from a consumer standpoint, I don't feel entirely comfortable with making cosmetic changes to the MRTK-Quest code without getting @provencher 's impression.

Reply:
Yeah I'll move forward with moving namespaces

Reply:
Updating the namespaces will also allow devs to import this side-by-side with the original MRTK-Quest extension without type collisions.